src/main/scala/com/databricks/spark/sql/perf/tpcds/Tables.scala

```diff
@@ -55,13 +56,21 @@ class Tables(sqlContext: SQLContext, dsdgenDir: String, scaleFactor: Int) extend
             sys.error(s"Could not find dsdgen at $dsdgen or /$dsdgen. Run install")
           }
 
-          // Note: RNGSEED is the RNG seed used by the data generator. Right now, it is fixed to 100.
-          val parallel = if (partitions > 1) s"-parallel $partitions -child $i" else ""
-          val commands = Seq(
-            "bash", "-c",
-            s"cd $localToolsDir && ./dsdgen -table $name -filter Y -scale $scaleFactor -RNGSEED 100 $parallel")
```

Comment:
Removing the -filter option breaks the way it is intended to be used with https://github.com/databricks/tpcds-kit.

If one wants an option to load from files instead of generating with -filter on the fly, it needs to be implemented as a (non-default) option, not replace the current behaviour.